system/modules/core/library/Contao/Encryption.php

```diff
@@ -93,9 +98,14 @@ public static function encrypt($varValue, $strKey=null)
 	 * @param string $strKey   An optional encryption key
 	 *
 	 * @return string The decrypted value
+	 *
+	 * @deprecated Deprecated since Contao 3.5, to be removed in Contao 5.
+	 *             Use the DCA load and save callbacks to implement your own encryption routines.
 	 */
 	public static function decrypt($varValue, $strKey=null)
 	{
+		@trigger_error('Using Encryption::decrypt() has been deprecated and will no longer work in Contao 5.0. Use the DCA load and save callbacks to implement your own encryption routines.', E_USER_DEPRECATED);
```

Comment:
I'm fine with @discordier's message. Though as mentioned in the respective ticket, people won't know where the message is coming from…